src/components/tile/tile-footer/tile-footer.component.js

```diff
@@ -8,7 +8,7 @@ const paddingPropTypes = filterStyledSystemPaddingProps(
   styledSystemPropTypes.space
 );
 
-const TileFooter = ({ variant, children, ...props }) => {
+export const TileFooter = ({ variant, children, ...props }) => {
```

Comment:
Similarly I don't think a named export for `TileFooter` is used anywhere, only the default one, so the `export` here may not be needed